it/src/test/java/org/corfudb/universe/scenario/TwoLinksFailureIT.java

```diff
@@ -82,6 +82,8 @@ public void twoLinksFailureTest() {
                     assertThat(table.get(String.valueOf(i))).isEqualTo(String.valueOf(i));
                 }
             });
+
+            corfuClient.shutdown();
```

Comment:
To prevent this, maybe we can implement corfuclient as autocloseable.
Just a suggestion. Not to be addressed right now.